### ISSUE TITLE
Pasting into textarea interferes with form submit?

### ISSUE THREAD

**Issue by skwp** [tiff/wysihtml5#391]
This is extremely strange. Paste into the wysihtml5 field, hit Save Changes on my form, it does nothing. Hit it again, it saves. The form is not submitting right after the paste. Anyone seen this? I am perplexed.

**Comment by mklb** [comment]
Yes I have - somehow its not submitting the data to my db when I paste content in the field.

**Comment by Verikon** [comment]
I got around this by attaching a listener to the paste event, placing a timeout function off that and setting value of the element (And in my case, a backbone model attribute). I don't have time right now to see if there's a more appropriate event fired specifically for this purpose, if it's just a browser quirk or throw a pull request up for at very least pushing the clipboard content into the "paste" event listener.
